src/components/Start/Start.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  RootState, toggleLoading, setTodos, getTodos,
+} from '../../store';
+import { getData } from '../../api';
+import { Todo, User } from '../../interfaces';
+
+type Props = {
+  loaderToggler: () => void;
+  todosSetter: (todos: Todo[]) => void;
+};
+
+const Start: React.FC<Props> = ({ loaderToggler, todosSetter }) => {
+  const getAllTodos = async () => {
+    loaderToggler();
+
+    const todos = await getData<Todo>('todos');
+    const users = await getData<User>('users');
+
+    const todosWithUsers: Todo[] = todos.map(todo => (
+      {
+        ...todo,
+        user: users.find(user => user.id === todo.userId),
```

Comment:
убирай знаки вопроса лишние а тут напиши вот так
```suggestion
        user: users.find(user => user.id === todo.userId) as User,
```

Reply:
они не лишние, тайпскрипт говорит так делать

Reply:
тебе нужно убрать знаки вопросов, у тебя такие данные что  User будет всегда, и тогда не будет проблемы в сорте, или есть другой вариант обработать ситуацию в которой юзер не нашел